website/blog/2022-05-25-async-indexing.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Asynchronous Indexing using Hudi"
+excerpt: "How to setup Hudi for asynchronous indexing"
+author: codope 
+category: blog
+---
+
+In one the previous posts, we discussed the design
+of [multi-modal index](https://www.onehouse.ai/blog/introducing-multi-modal-index-for-the-lakehouse-in-apache-hudi)
+, a serverless and high-performance indexing subsystem to the Lakehouse
+architecture to boost the read and write latencies. In this blog, we discuss the
+design of asynchronous indexing, a first of its kind, to support index building
+alongside regular writers without blocking ingestion.
+
+<!--truncate-->
+
+## Background
+
+Built on top of cheap storage and open file formats, the Hudi stack is more than
```

Comment:
Should I add the hudi stack figure from https://hudi.apache.org/blog/2021/07/21/streaming-data-lake-platform/#hudi-stack